src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCodeV1.java

```diff
@@ -204,7 +204,8 @@ public enum ErrorCodeV1 {
   TABLE_MISSING_PARTITIONING_KEYS("Key needs to be provided as part of partitioning definition"),
   TABLE_COLUMN_DEFINITION_MISSING("Column definition is missing for the provided key field"),
   TABLE_COLUMN_TYPE_UNSUPPORTED("Unsupported column types"),
-  TABLE_COLUMN_UNKNOWN("Column unknown");
+  TABLE_COLUMN_UNKNOWN("Column unknown"),
```

Comment:
Use the APiException pls, there is an existing for unknown column. Replace the TABLE_INVALID_FILTER with something more meaninfull,  COMPARISION_ON_DURATION_NOT_SUPPORTED

Reply:
please refer to this comment.
https://github.com/stargate/data-api/pull/1448#discussion_r1783614162